tern/analyze/common.py

```diff
@@ -473,8 +474,11 @@ def consolidate_commands(command_list):
                 new_list.append(second)
             else:
                 if not first.merge(second):
-                    command_list.append(first)
```

Comment:
append(first) here may cause two or more same Command objects in the `command_list`.
It seems more reasonable to append `second` since we have popped `second` from the list, if we cannot merge `second` into `first`, we need to keep `second` in the list instead of `first` for next while loop to analyze it.